Docs: Updates Changelog for 6.6.0 (#21753)
grafana/grafana: CHANGELOG.md

> # 6.6.0 (2020-01-27)

Reviewer: Would add it under a new `# 6.7.0 (unreleased)` section instead since introduced after branching of 6.6.x release branch. Either keep notice or add it under breaking changes section.

## Notice about upcoming changes in backendSrv for plugin authors

> In our mission to migrate away from AngularJS to React we have removed all AngularJS dependencies in the core data retrieval service `backendSrv`. This change is already in master and will be introduced in the next `major` Grafana release.

Reviewer 2: ```suggestion
In our mission to migrate away from AngularJS to React we have removed all AngularJS dependencies in the core data retrieval service `backendSrv`.
```

This is information is tied to a specific release of Grafana, it being in master should be self evident.

Author: We're just making sure that everyone understands that this is an upcoming change that will be in the next major release.

Reviewer 3: ```suggestion
In our mission to migrate away from AngularJS to React we are removing all AngularJS dependencies in the core data retrieval service `backendSrv`. This change will be introduced in Grafana 7.0.0.
```

?

Reviewer 3: (assuming you mean major in the semver kind of way)

Author: We're not sure if this is going to be released in 6.7 or 7.0 @sakjur @xlson  so that's why we wrote this open ended text

Reviewer 3: Ah, major release implies `7.0.0` to me (first digit is major, second is minor, and third is patch).

```suggestion
In our mission to migrate away from AngularJS to React we are removing all AngularJS dependencies in the core data retrieval service `backendSrv`. This change will be introduced in the next regular Grafana release.
```

Maybe? And then down below something like

```
You can test your plugin with the `master` branch version of Grafana.
```

Reviewer 2: Ah, in that case I think we should put it under 7.0.0 (unreleased). We can move it to the 6.7.0 section if we end up releasing a 6.7.0 with it.

Removing the AngularJS dependencies in `backendSrv` has the unfortunate side effect of AngularJS digest no longer being triggered for any request made with `backendSrv`. Because of this, external plugins using `backendSrv` directly may suffer from strange behaviour in the UI.

> To remedy this issue as a plugin author you need to trigger the digest after a direct call to `backendSrv`. 

Reviewer 2: ```suggestion
To remedy this issue, as a plugin author you need to trigger the digest after a direct call to `backendSrv`. 
```

I think there should be a comma.
